lib/snail.rb

```diff
@@ -35,6 +35,7 @@ class UnknownAttribute < ArgumentError; end
     :locality   => :city,
     :state      => :region,
     :province   => :region,
+    :county     => :region,
```

Comment:
yeah this aliasing may not be necessary, but Irish counties are used equivalently to US states in mailing addresses and US counties aren't really used in mailing addresses, so I threw it in there.

open to pulling that part out though.